### ISSUE TITLE
setup-python 3.x fails in CML image

### ISSUE THREAD

**Issue by DavidGOrtega** [iterative/cml#386]
```sh
Version 3.x was not found in the local cache
Version 3.x is available for downloading
Download from "https://github.com/actions/python-versions/releases/download/3.9.1-96692/python-3.9.1-linux-18.04-x64.tar.gz"
Extract downloaded archive
/bin/tar xz --warning=no-unknown-keyword -C /__w/_temp/30c08b87-7a0d-4e73-abfc-459cf22b29ef -f /__w/_temp/0b5a57d7-bcf6-41c4-bfbd-e3213c5a7683
Execute installation script
Check if Python hostedtoolcache folder exist...
Creating Python hostedtoolcache folder...
Create Python 3.9.1 folder
Copy Python binaries to hostedtoolcache folder
Create additional symlinks (Required for the UsePythonVersion Azure Pipelines task and the setup-python GitHub Action)
Upgrading PIP...
Looking in links: /tmp/tmpotuw24aa
Requirement already satisfied: setuptools in /__t/Python/3.9.1/x64/lib/python3.9/site-packages (49.2.1)
Requirement already satisfied: pip in /__t/Python/3.9.1/x64/lib/python3.9/site-packages (20.2.3)
Collecting pip
Downloading pip-21.0.1-py3-none-any.whl (1.5 MB)
Installing collected packages: pip
Successfully installed pip-21.0.1
Create complete file
Error: Version 3.x with arch x64 not found
The list of all available versions can be found here: https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json
```

**Comment by DavidGOrtega** [comment]
https://github.com/actions/setup-python/issues/96#issuecomment-779224897

**Comment by DavidGOrtega** [comment]
closed via #406
